sdk/core/core/inc/az_version.h

```diff
@@ -15,9 +15,21 @@
 #ifndef _az_VERSION_H
 #define _az_VERSION_H
 
-int az_core_version_get_major();
-int az_core_version_get_minor();
-int az_core_version_get_patch();
-const char* az_core_version_get_prerelease();
+// @brief AZ_SDK_VERSION_STRING Version string for telemetry
```

Comment:
You don't need to repeat the name of the macro/method in the brief. It shows up as redundant. See https://github.com/Azure/azure-sdk-for-c/pull/583#discussion_r409809998 / https://github.com/Azure/azure-sdk-for-c/issues/589

```suggestion
// @brief The SDK version string used for telemetry.
```

Reply:
Updated